
laser_topic: "/hsrb/base_scan"
pc_topic: "/hsrb/head_rgbd_sensor/depth_registered/rectified_points"
min_height_: 0.0
max_height_: 1.0

#voxel size
leaf_size : 0.01

#z filter length
z_filter_min : 0.0
z_filter_max: 1.5





